### PR TITLE
Integration1 t6f2 53

### PR DESCRIPTION
**Pull request by nala-he**
Hi guys, I built upon Am's critic review portion with two screens (editing and display screen). Currently you can write reviews, submit, and see the updated review list displayed as a critic user. To switch to the average user, comment out line 20 and use line 22 as the loggedIn user. We will update all hardcoded part after setting up servers and login/signup. I also revised the style name to ttr, added the business profile (use this path to access for now: "/profile/business/123").

### CODE REVIEW

File: src/components/profile/business-profile.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <img src={`/images/${restaurant.bannerPicture}`}
+                 className="ttr-border-radius ttr-banner-width mt-3" height={200}/>
+            <img className="ttr-portrait position-absolute start-0 ms-5"
+                 src={`/images/${restaurant.profilePicture}`}/>
+        </div>
+        <div className="m-3 position-relative">
+                <span className="h5 fw-bolder">{restaurant.name} {restaurant.handle}
+                </span><br/>
+            {/*Profile edit button*/}
+            <Link to="./edit-restaurant">
```
Comment (MaceeQi) [inline src/components/profile/business-profile.js]: I think Yena implemented the path to edit business profile as /restaurant/:rid/edit. Let's agree on a path before merging

**Review by MaceeQi — CHANGES_REQUESTED** [review]
Left a quick comment regarding the path the edit button navigates to since there is discrepancy with pull request #6. Let's agree on a path for editing business profile before merging. Otherwise, everything else looked good!

File: src/components/profile/business-profile.js (hunk 1 above)
Comment (nala-he) [reply src/components/profile/business-profile.js]: Sounds good.

Comment (nala-he) [reply src/components/profile/business-profile.js]: I made the revision on my path according to the conversation in request#6.

**Comment by nala-he** [comment]
I made the revision on my path and link according to the conversation in request#6. See the bottom commit record.

**Comment by MaceeQi** [comment]
Thanks for the changes, Yutong. I ran the branch locally and noticed that at path /profile/restaurant/:rid and /profile/restaurant/:rid/edit, both the Profile and Restaurant are highlighted in the navbar. Was that intentional?

**Comment by nala-he** [comment]
> Thanks for the changes, Yutong. I ran the branch locally and noticed that at path /profile/restaurant/:rid and /profile/restaurant/:rid/edit, both the Profile and Restaurant are highlighted in the navbar. Was that intentional?

Oh, that's not intentional. But I think it's because we are using both profile and restaurant keywords in the path for the nav bar button and it highlighted both. In this case, can we use /profile/business/:rid to keep it easy?

**Review by bovornc — APPROVED** [review]
cloned and ran it: everything looks good on my end!

**Comment by MaceeQi** [comment]
Perfect! Good to merge :)

**Review by MaceeQi — APPROVED** [review]
Perfect! Good to merge :)

**Pull request merged**